src/plugins/koops_event.conf

```diff
@@ -30,23 +30,23 @@ EVENT=post-create type=Kerneloops remote!=1
 #EVENT=report_Kerneloops type=Kerneloops
         reporter-kerneloops
 
-EVENT=report_Bugzilla type=Kerneloops
+EVENT=report_Bugzilla type=Kerneloops mce!=non-fatal mce!=fatal
         reporter-bugzilla -b \
                 -F /etc/libreport/plugins/bugzilla_format_kernel.conf
 
 # Send micro report
-EVENT=report_uReport type=Kerneloops
+EVENT=report_uReport type=Kerneloops mce!=non-fatal mce!=fatal
```

Comment:
Another option would be to turn of the error message in `abrt-action-notify`. We can say that if there is no AutoReportEvent (`/etc/abrt.conf`) enabled for the notified problem we skip this notification channel. The error message should be printed out in debug mode.